src/MoBi.Presentation/Tasks/Edit/EditTaskForContainer.cs

```diff
@@ -156,12 +162,38 @@ private IParameter createParameter(IndividualParameter individualParameter)
       {
          var parameterToAdd = _individualParameterToParameterMapper.MapFrom(individualParameter);
 
+         if (parameterToAdd.Name.Equals(Constants.Distribution.PERCENTILE) && individualParameter.Value.HasValue)
+            setValuesForPercentile(parameterToAdd, individualParameter.Value.Value, individualParameter.Dimension);
+         else
+            setValues(individualParameter, parameterToAdd);
+
+         // NaN formulas do not need to be serialized
+         if (parameterToAdd.Formula is ConstantFormula constantFormula && double.IsNaN(constantFormula.Value)) 
+            parameterToAdd.Formula = null;
+
+         return parameterToAdd;
+      }
+
+      private static void setValues(IndividualParameter individualParameter, IParameter parameterToAdd)
+      {
          if (individualParameter.Formula != null)
             parameterToAdd.Formula = individualParameter.Formula;
          if (individualParameter.Value.HasValue)
             parameterToAdd.Value = individualParameter.Value.Value;
```

Comment:
A lot of the code for this PR is related to finding the right way to construct this PercentileParameter in such  way that it makes sense and serializes/deserializes properly

We think after some discussion that this line of code needs to be replaced with a line that creates a constant formula instead of setting a value, and also make sure the IsFixedValue is set to false.